schemas/2019-06-01/Microsoft.Storage.json

```diff
@@ -7,34 +7,22 @@
     "storageAccounts": {
       "type": "object",
       "properties": {
-        "name": {
-          "type": "string",
-          "minLength": 3,
-          "maxLength": 24,
-          "description": "The name of the storage account within the specified resource group. Storage account names must be between 3 and 24 characters in length and use numbers and lower-case letters only."
-        },
-        "type": {
```

Comment:
It looks type,tag,sku are missed from storage account? will the template still work after the change?